src/meinberlin/meinberlin/static/stylesheets/scss/components/_badges_theme.scss

```diff
@@ -1,11 +1,11 @@
 /*doc
 ---
-title: badges
-name: badges
+title: badges MeinBerlin
+name: badges-mainberlin
 category: Type
 ---
 
-Badges for labeling if a proposal has been realized or not.
+Badge modifiers for labeling if a proposal has been realized or not.
```

Comment:
You are right, these are modifiers. But their names suggest that they are states. Could you please adapt the names?